src/azure/Elastic.Apm.Azure.Functions/AzureFunctionsDiagnosticListener.cs

```diff
@@ -27,7 +27,7 @@ internal class AzureFunctionsDiagnosticListener : DiagnosticListenerBase
 
 	protected override void HandleOnNext(KeyValuePair<string, object> kv)
 	{
-		Context.Logger.Trace()?.Log($"'{nameof(AzureFunctionsDiagnosticListener)}.{nameof(HandleOnNext)}': {kv.Key}");
+		Context.Logger.Trace()?.Log($"'{nameof(AzureFunctionsDiagnosticListener)}.{nameof(HandleOnNext)}': {{DiagnosticsEventName}}", kv.Key);
```

Comment:
Should we move the concatenation of the class and method name and also pass that as an argument here, too?

Reply:
Since its a constant I think it makes sense to leave it in like it is.

Reply:
Yeah, fair point.